Middlewares/signup.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require("jsonwebtoken");
```

Comment:
you can use salt + bcrypt to hash your passwords

Reply:
I would make changes in the user router soon. Its under progress